aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/OutboxMutationFailedEvent.java

```diff
@@ -35,16 +35,16 @@
  */
 public final class OutboxMutationFailedEvent<M extends Model>
         implements HubEvent.Data<OutboxMutationFailedEvent<M>> {
-    private final List<GraphQLResponse.Error> errors;
+    private final MutationErrorType errorType;
     private final MutationType operation;
     private final M model;
 
```

Comment:
As below, you're probably going to want to make `modelName` an explicit property of the shape, as opposed to trying to pull it from the `model` in the accessor.